ImportError on test_trame.py

**Describe the bug**

Getting error follow [I want to help](https://github.com/pyinstaller/pyinstaller-hooks-contrib?tab=readme-ov-file#i-want-to-help) guide.

Tried clean `pyenv`, and I've contributed before using same machine / environment.

**To Reproduce**

In terminal:
```
 pyenv shell 3.12.2
 pyenv virtualenv hooks-contrib-3.12.2
 python -m pip install -e .
 python -m pip install -r requirements-test.txt
 python -m pip install flake8 pyinstaller\n
 pytest
```

Error:
```
====================================== test session starts =======================================
platform darwin -- Python 3.12.2, pytest-8.3.3, pluggy-1.5.0 -- /Users/dave/.pyenv/versions/3.12.2/bin/python
cachedir: .pytest_cache
rootdir: /Users/dave/code/pyinstaller-hooks-contrib
configfile: setup.cfg
plugins: drop-dup-tests-1.0.0, anyio-4.4.0, timeout-2.3.1, xdist-3.6.1
collected 490 items / 1 error / 488 deselected / 2 selected                                      

============================================= ERRORS =============================================
______________________________ ERROR collecting tests/test_trame.py ______________________________
ImportError while importing test module '/Users/dave/code/pyinstaller-hooks-contrib/tests/test_trame.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.pyenv/versions/3.12.2/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_trame.py:16: in <module>
    from PyInstaller.utils.hooks import check_requirement
E   ImportError: cannot import name 'check_requirement' from 'PyInstaller.utils.hooks' (/Users/dave/.pyenv/versions/3.12.2/lib/python3.12/site-packages/PyInstaller/utils/hooks/__init__.py)
======================================== warnings summary ========================================
../../.pyenv/versions/3.12.2/lib/python3.12/site-packages/PyInstaller/lib/modulegraph/modulegraph.py:16
  /Users/dave/.pyenv/versions/3.12.2/lib/python3.12/site-packages/PyInstaller/lib/modulegraph/modulegraph.py:16: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    import pkg_resources

../../.pyenv/versions/3.12.2/lib/python3.12/site-packages/setuptools/_distutils/ccompiler.py:855
  /Users/dave/.pyenv/versions/3.12.2/lib/python3.12/site-packages/setuptools/_distutils/ccompiler.py:855: DeprecationWarning: includes is deprecated
    warnings.warn("includes is deprecated", DeprecationWarning)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
==================================== short test summary info =====================================
ERROR tests/test_trame.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================== 488 deselected, 2 warnings, 1 error in 1.01s ==========================
```

**Expected behavior**

`master` always works in clean env.


**Desktop (please complete the following information):**
 - OS: macOS
 - Python Version: 3.12.2
 - Version of `pyinstaller-hooks-contrib`: 2024.8
 - Version of PyInstaller 5.13.1

**Additional context**

`test_trame.py` came in [just after](https://github.com/pyinstaller/pyinstaller-hooks-contrib/commit/46b318260d89f1da5f95962d262b09f25fb0d56c) my [last contribution](https://github.com/pyinstaller/pyinstaller-hooks-contrib/commit/46b318260d89f1da5f95962d262b09f25fb0d56c).

--- a/tests/test_trame.py
+++ b/tests/test_trame.py
@@ -13,13 +13,13 @@
 import pytest
 
 from PyInstaller.utils.tests import importorskip
-from PyInstaller.utils.hooks import check_requirement
+from PyInstaller.utils.hooks import is_module_satisfies
 
 
 # Pretty much all tests here require `trame.app module` from core `trame` dist, so skip the tests if it is not
 # available (installing other `trame-*` dists does not seem to install `trame` dist itself). We could equivalently
 # check if `trame.app` is importable, but that would require an isolated check.
-pytestmark = pytest.mark.skipif(not check_requirement('trame'), reason="Core 'trame' distribution is not installed.")
+pytestmark = pytest.mark.skipif(not is_module_satisfies('trame'), reason="Core 'trame' distribution is not installed.")
 
 
 @importorskip("trame")

